packages/angular/cli/commands/new.md

```diff
@@ -0,0 +1,5 @@
+Creates and initializes a new Angular app that is the default project for a new workspace.
+
+* The new workspace folder is given the specified project name. The new app is created at the top level of the workspace folder.
```

Comment:
The new app is in the src folder. That's where all the app-specific source code and configuration files live.